app/policies/schools/course_policy.rb

```diff
@@ -39,6 +39,7 @@ def curriculum?
     end
 
     alias evaluation_criteria? curriculum?
+    alias clone? curriculum?
```

Comment:
Only users who can create a course should be able to clone a course.

```suggestion
    alias clone? index?
```